app/models/dao.py

```diff
@@ -97,30 +109,34 @@ def save_user_identification(id_user, identity):
                 values
                 ('{identity}', '{id_user}')
                 """
-                cursor.execute(query_identity)
             elif len(identity) == 14 or re.match(r"^\d{2}\.\d{3}\.\d{3}\/\d{4}\-\d{2}$", identity):
                 query_identity = f"""
                 insert into info_user_cnpj(cnpj, id_user)
                 values
                 ('{identity}', '{id_user}')
                 """
-                cursor.execute(query_identity)
+                print('cnpj')
```

Comment:
Cuidado com o print de informações sensíveis e PII, para debug blz, mas cuidado quando for subir a produção